### PR TITLE
Named exports from filter module

### PR DESCRIPTION
**Pull request by marcjansen**
Please review.

### CODE REVIEW

File: src/ol/format/filter.js [hunk 1]
```diff
@@ -28,9 +27,9 @@ const _ol_format_filter_ = {};
  * @returns {!ol.format.filter.And} `<And>` operator.
  * @api
  */
-_ol_format_filter_.and = function(conditions) {
+export const and = function(conditions) {
```
Comment (fredj) [inline src/ol/format/filter.js]: could you change it to `export function and(conditions) {`, like it's done elsewhere?

Comment (marcjansen) [reply src/ol/format/filter.js]: Of course. Will do later.

**Comment by marcjansen** [comment]
Updated as suggested, @fredj

**Comment by marcjansen** [comment]
Thanks for the reviews.

**Pull request merged**